packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/src/index.ts

```diff
@@ -1,6 +1,14 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+import {
+  _Metrics,
+  _PagedTestFileInfo,
+  _PagedTest,
+  _PagedTestRun,
+  _PagedTestProfile,
+  _PagedTestProfileRun,
+} from "./models/models.js";
```

Comment:
this doesn't seem correct as models/models.ts has moved out of the subclient folder.